pages/search.vue

```diff
@@ -584,57 +586,93 @@ export default {
 
 .search-controls {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
+  gap: var(--spacing-card-md);
+
+  .sort-controls {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    gap: var(--spacing-card-md);
+    flex-wrap: wrap;
+
+    .labeled-control {
+      flex: 1;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      flex-wrap: wrap;
+      gap: 0.5rem;
+
+      .labeled-control__label {
+        white-space: nowrap;
+      }
+    }
+  }
 
   .iconified-input {
-    margin-left: 6px;
+    flex: 1;
 
     input {
-      width: calc(100% + 8px);
+      min-width: 15rem;
+      width: 100%;
     }
   }
 }
 
 .search-controls__sorting {
-  min-width: 15rem;
+  min-width: 14rem;
 }
 
 .labeled-control__label,
 .labeled-control__control {
   display: block;
-  margin: 0.5rem 0;
 }
 
 .no-results {
   text-align: center;
 }
 
-@media (min-width: 1024px) {
-  .sidebar-menu {
-    display: block;
-    margin-top: 0;
+@media (min-width: 700px) {
```

Comment:
```suggestion
@media (min-width: 850px) {
```
Between 700 and 850px, the per-page kind of glitches out